pkg/sql/sem/builtins/aggregate_builtins.go

```diff
@@ -400,42 +400,10 @@ var aggregates = map[string]builtinDefinition{
 			tree.VolatilityImmutable,
 		),
 	),
-	"st_union": makeBuiltin(
-		tree.FunctionProperties{
-			Class:                   tree.AggregateClass,
-			NullableArgs:            true,
-			AvailableOnPublicSchema: true,
-		},
-		makeAggOverload(
-			[]*types.T{types.Geometry},
-			types.Geometry,
-			func(
-				params []*types.T, evalCtx *tree.EvalContext, arguments tree.Datums,
-			) tree.AggregateFunc {
-				return &stUnionAgg{}
-			},
-			infoBuilder{
-				info: "Applies a spatial union to the geometries provided.",
-			}.String(),
-			tree.VolatilityImmutable,
-		),
-	),
-	"st_collect": makeBuiltin(
-		tree.FunctionProperties{
-			Class:                   tree.AggregateClass,
-			NullableArgs:            true,
-			AvailableOnPublicSchema: true,
-		},
-		makeAggOverload(
-			[]*types.T{types.Geometry},
-			types.Geometry,
-			newSTCollectAgg,
-			infoBuilder{
-				info: "Collects geometries into a GeometryCollection or multi-type as appropriate.",
-			}.String(),
-			tree.VolatilityImmutable,
-		),
-	),
+	"st_union":      makeSTUnionBuiltin(),
+	"st_memunion":   makeSTUnionBuiltin(),
+	"st_collect":    makeSTCollectBuiltin(),
+	"st_memcollect": makeSTCollectBuiltin(),
```

Comment:
you can re-use the alias stuff for these: https://github.com/cockroachdb/cockroach/blob/master/pkg/sql/sem/builtins/geo_builtins.go#L4622

also https://github.com/cockroachdb/cockroach/pull/53556 is in the queue and you'll probably have to rebase over the top of them.

thanks!

Reply:
oh nvm you can't :P

Reply:
I'm off to bed shortly, so feel free to rebase, otherwise I'll get to it tomorrow.